ios/RNCSafeAreaShadowView.m

```diff
@@ -6,6 +6,7 @@
 #import "RNCSafeAreaViewEdges.h"
```

Comment:
Is this file used anymore? We might be able to delete it

Reply:
You mean delete header import? RNCSafeAreaViewEdges.h is where RNCSafeAreaViewEdges struct lives

Reply:
https://github.com/th3rdwave/react-native-safe-area-context/blob/main/ios/RNCSafeAreaViewEdges.h and https://github.com/th3rdwave/react-native-safe-area-context/blob/main/ios/RNCSafeAreaViewEdges.m - does these still get used?

Reply:
Sorry - I see what you mean. Maybe we could rename it RNCSafeAreaViewEdgeMode - so it matches the type name